README.md

```diff
@@ -1,24 +1,19 @@
 # isar-turtlebot
 ISAR implementation for the Turtlebot3 Waffle Pi.
 
-# Installation for development
+[ISAR](https://github.com/equinor/isar) - Integration and Supervisory control of Autonomous Robots - is a tool for integrating robot applications into Equinor systems. Through the ISAR API you can send command to a robot to do missions and collect results from the missions.
 
-For local development, please fork the repository. Then, clone and install in the repository root folder:
+Running the full ISAR system requires an installation of a robot which satisfies the required [interface](https://github.com/equinor/isar/blob/main/src/robot_interface/robot_interface.py). isar-turtlebot is an ISAR implementation of the Turtlebot3 Waffle Pi, which enables simualtions of ISAR.
```

Comment:
Do we need to focus it on the simulation, or shall the turtlebot repository also cover the physical turtlebot?

Reply:
Good point, maybe that last part can be erased?